js/config/syllables.js

```diff
@@ -31,6 +35,419 @@ module.exports = function() {
 			{ word: "i.e", syllables: 2}
 		],
 		vowels: 'aeiouy'
-	};
+	},
+	nl: {
```

Comment:
Please consider sorting the exclusion words in alphabetical order. This makes both reading the config and finding specific words, a lot easier.